app/pages/capability-selector/template.njk

```diff
@@ -0,0 +1,30 @@
+{% extends 'includes/layout.njk' %}
+{% from './components/macro.njk' import capabilitySelector %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/button/macro.njk' import button %}
+
+{% block body %}
+  <div  class="nhsuk-width-container">
+    <div class="nhsuk-u-padding-top-5" data-test-id="go-back-link">
+      {{ backLink({
+        "href": "/solutions",
+        "text": "Go back to previous page"
+      }) }}
+    </div>
+    <h1 class="nhsuk-u-margin-top-6 nhsuk-u-margin-bottom-4" data-test-id="capability-selector-page-title">{{ title }}</h1>
+    <div class="nhsuk-grid-row">
+      <h2 class="nhsuk-u-font-size-22 nhsuk-u-padding-bottom-5 nhsuk-grid-column-two-thirds" data-test-id="capability-selector-page-description">{{ description }}</h2>
+      {{ capabilitySelector({
+        capabilities: capabilities
+      }) }}
+    </div>
+    <div data-test-id="capability-selector-continue-button">
+      {# TODO: change href of button #}
```

Comment:
we going to change this in the next ticket? when we do the post?

Reply:
yep I thought so or we could do it in it's own ticket